src/assets/toolkit/styles/toolkit.scss

```diff
@@ -2,7 +2,13 @@
  * Toolkit styles
  */
 
+@import "./functions";
+@import "../../../data/toolkit.yml";
 
 body {
 	font-family: sans-serif;
 }
+
+.f-menu a:hover {
```

Comment:
Remove this example. Helpful to see the demo, but maybe fits better as a comment in the `functions` dir.